macos/Sources/Features/QuickTerminal/QuickTerminalController.swift

```diff
@@ -67,6 +67,9 @@ class QuickTerminalController: BaseTerminalController {
         // The quick window is not restorable (yet!). "Yet" because in theory we can
         // make this restorable, but it isn't currently implemented.
         window.isRestorable = false
+        
+        // Propagate `focus-follows-mouse` options to the quick terminal.
+        window.focusFollowsMouse = ghostty.config.focusFollowsMouse
```

Comment:
BaseTerminalController has a windowDidLoad you can add this to. It might make sense to add a function there to call on config change too (notification is already there too).